cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetectorManagerTest.java

```diff
@@ -522,7 +525,7 @@ public void testExecutionInProgress() throws InterruptedException {
     KafkaCruiseControl mockKafkaCruiseControl = EasyMock.mock(KafkaCruiseControl.class);
     Properties props = KafkaCruiseControlUnitTestUtils.getKafkaCruiseControlProperties();
     KafkaCruiseControlConfig kafkaCruiseControlConfig = new KafkaCruiseControlConfig(props);
-    EasyMock.expect(mockKafkaCruiseControl.config()).andReturn(kafkaCruiseControlConfig).times(2);
+    EasyMock.expect(mockKafkaCruiseControl.config()).andReturn(kafkaCruiseControlConfig).atLeastOnce();
```

Comment:
Can we make the number of calls specific?

Reply:
Reverted this change in all files